docs/setup/configuring-reporting.asciidoc

```diff
@@ -231,3 +205,44 @@ For more information, see {ref}/notification-settings.html#ssl-notification-sett
 Once you've enabled SSL for {kib}, all requests to the reporting endpoints must include valid credentials.
 
 For more information on sharing reports, direct links, and more, refer to <<reporting-getting-started, Reporting and sharing>>.
+
+[float]
+[[install-reporting-packages]]
+=== Install the dependencies for the headless browser
+
+If using PNG/PDF {report-features}, make sure the {kib} server operating system has the appropriate packages installed for the distribution.
```

Comment:
Since this section only applies to PDF/PNG reports, it makes sense to move it to the bottom of the page.